examples/places/place-with-taxonomy.yaml

```diff
@@ -0,0 +1,55 @@
+---
+id: overture:places:place:77
+type: Feature
+geometry:
+  type: Point
+  coordinates: [-97.7431, 30.2672]
+properties:
+  basic_category: casual_eatery
+  taxonomy:
+    primary: gas_station_sushi
+    hierarchy:
+      - food_and_drink
+      - restaurant
+      - casual_eatery
+      - gas_station_sushi
+    alternates:
+      - gas_station
+      - sushi_restaurant
+  categories:
+    primary: gas_station_sushi
+    alternates:
+      - gas_station
+      - sushi_restaurant
```

Comment:
This example makes it super clear that apart from clearer documentation the main thing we're adding is the `hierarchy` - everything else was already there...

I'm curious then: was not having the primary hierarchy considered a major usability obstacle?

Reply:
Got it. I now have the whole picture in my head, at least temporarily!

Hopefully our messaging on this is good :)